docker-compose.yaml

```diff
@@ -1,7 +1,15 @@
 services:
+  nginx:
```

Comment:
Should we somehow make the usage of the nginx a standard for developers using the compose setup? meaning only the nginx should be access from the outside and the webapp can only be access through the docker network?

Reply:
Sounds appropriate but didn't want to disrupt to many things at the same time. Don't know if there are other implications. How about leaving it optional for now and we discuss it in a tech daily if anybody objects?

Reply:
Fine for me